docs/proposals/1374-multi-cluster-inference/README.md

```diff
@@ -0,0 +1,401 @@
+       - Connect to remote parents, e.g. IGs, of the exported InferencePool (Parent Mode).
+4. **Data Path:**
+   The data path is dependant on the export mode selected by the user.
+   - Endpoint Mode: Client → local IG → (make scheduling decision) → local/remote EPP → selected model server endpoint → response.
+   - Parent Mode: Client → local IG → (make scheduling decision) → local EPP/remote parent → remote EPP → selected model server endpoint → response.
+
+### InferencePoolImport Naming
+
+The exporting controller will create an InferencePoolImport resource using the exported InferencePool namespace and name. A cluster name entry in
+`inferencepoolimport.statu.clusters[]` is added for each cluster that exports an InferencePool with the same ns/name.
+
+### InferencePool Selection
+
+InferencePool selection is implementation-specific. The following are examples of how an IG may select one exported InferencePool over another:
+
+- **Metrics-based:** Scrape EPP-exposed metrics (e.g., ready pods) to bias InferencePool choice.
+- **Active-Passive:** Basic EPP readiness checks (gRPC health).
+
+**Note:** When an exported InferencePool is selected by an IG, standard EPP semantics are used to select endpoints of that pool.
+
+### API Changes
+
+#### Export Annotation
+
+The following annotation is being proposed to indicate the desire to export the InferencePool to member clusters of a ClusterSet.
+
+The `inference.networking.x-k8s.io/export` annotation key indicates a desire to export the InferencePool:
+
+```yaml
+inference.networking.x-k8s.io/export: "<value>"
+```
+
+Supported Values:
+
+- `ClusterSet` – export to all members of the current [ClusterSet](https://multicluster.sigs.k8s.io/api-types/cluster-set/).
+
+**Note:** Additional annotations, e.g. region/domain scoping, filter clusters in the ClusterSet, routing mode configuration, etc. and
+potentially adding an InferencePoolExport resource may be considered in the future.
+
+#### InferencePoolImport
+
+A cluster-local, controller-managed resource that represents an imported InferencePool. It primarily communicates a relationship between an exported
+InferencePool and the exporting cluster name. It is not user-authored; status carries the effective import. Inference Platform Owners can reference the InferencePoolImport, even if the local cluster does not have an InferencePool. In the context of Gateway API, it means that an HTTPRoute can be configured to reference an InferencePoolImport to route matching requests to remote InferencePool endpoints. This API will be used almost exclusively for tracking endpoints,
+but unlike MCS, we actually have two distinct sets of endpoints to track:
+
+1. Endpoint Picker Extensions (EPPs)
+2. InferencePool parents, e.g. Gateways
+
+Key ideas:
+
+- Map exported InferencePool to exporting cluster name.
+- Name/namespace sameness with the exported InferencePool (avoids extra indirection).
```

Comment:
I think Istio will need to require namespace sameness on the EPP as well since we don't pick the cluster first before load balancing (so we couldn't support a remote inference pool having a different EPP)

Reply:
Same as https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/1374#discussion_r2361250873